src/frontend/js/components/CourseProductsList/index.spec.tsx

```diff
@@ -119,11 +123,8 @@ describe('CourseProductsList', () => {
     });
 
     // - It should render one <CourseProductItem /> per product
-    expect(screen.getAllByRole('heading', { level: 2 })).toHaveLength(course.products.length);
-
+    expect(container.querySelectorAll('.product-widget')).toHaveLength(course.products.length);
```

Comment:
Oh okay, will change.

I changed that so that the mocked stuff better matched the actual component. So that if one day we would stop mocking the component for whatever reason, it would certainly result in less testing code change in this file.